lib/ccznp.js

```diff
@@ -372,7 +375,7 @@ CcZnp.prototype._mtIncomingDataHdlr = function (err, data) {
             ind: cmd,
             data: result
         };
-
+console.log(rxEvt,msg)
```

Comment:
I think these `console.log` (all) have to be removed.

Reply:
Thank you, forgot to push.